### ISSUE TITLE
[REQ] Support linting the `%` character

### ISSUE THREAD

**Issue by ooker777** [sneaky-foxes/obsidian-safe-filename-linter#17]
Notes whose filename containing the percent character `%` [doesn't do well when being published to the web](https://stackoverflow.com/q/7449588/3416774). I want to automatically convert it to `％`. Is it possible?

**Comment by lomky** [comment]
For web publishing, a system should be [URL encoding](https://en.wikipedia.org/wiki/Percent-encoding) all note names. See [URL Encoding Percent Character](https://en.wikipedia.org/wiki/Percent-encoding#Percent_character). To support note names that can be published to the web without URL encoding feels out of scope to me.

Can you tell me more about what problem you are running into in particular? It might be that you need to submit a bug report to your publishing software or add encoding if its your own.

**Comment by ooker777** [comment]
No, I am dealing with the publishing software already. But it will be neat if I can lint the filename right at the source, i.e. Obsidian. Hence I suggest to add support for that character in this plugin.

**Comment by sigrunixia** [comment]
I'd like to add my +1 for a different reason: Some roms and filesystems on Android do not support `%` as an allowed character. I get several emails a week in the Obsidian support box, and occasionally the problem special character is a `%` in the filename.
